husky_components/input/input-type.js

```diff
@@ -9,8 +9,8 @@
  */
 
 define([
-    'type/default'
-], function(Default) {
+    'type/default','services/husky/url-validator'
+], function(Default, UrlValidator) {
```

Comment:
Didn't we agree to start services with lower letters, since they are not constructors?